admin/services/check-form.service.js

```diff
@@ -126,6 +126,24 @@ const checkFormService = {
       const promises = modifiedCheckWindows.map(cw => { cw.save() })
       return Promise.all(promises)
     }
+  },
+  /**
+   * Append check windows name(s) and canDelete to check form object.
+   * @param checkWindows
+   * @returns {Promise.<void>}
+   */
+  objectFormattingForSinglePage: async (checkWindows) => {
+    let data = {
+      checkWindowsName: [],
+      canDelete: true
+    }
+    checkWindows.forEach(cw => {
+      data.checkWindowsName.push(' ' + cw.checkWindowName)
+      if (cw.checkStartDate <= Date.now()) {
```

Comment:
It seems a little weird to have business logic in a formatting function.

Reply:
You would also want use moment.utc() here instead